# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsndfile/regtest-sqlite.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libsndfile-1.0.17-orig/configure.ac libsndfile-1.0.17/configure.ac
--- libsndfile-1.0.17-orig/configure.ac	2008-03-06 01:35:07.000000000 +0000
+++ libsndfile-1.0.17/configure.ac	2008-03-06 01:36:40.000000000 +0000
@@ -268,6 +268,7 @@
 	fi
 
 AC_DEFINE_UNQUOTED([HAVE_SQLITE3],$HAVE_SQLITE3,[Set to 1 if you have libsqlite3.])
+AM_CONDITIONAL(HAVE_SQLITE3, [test "x$ac_cv_sqlite3" = "xyes"])
 
 #====================================================================================
 # Determine if the processor can do clipping on float to int conversions.
diff -Nur libsndfile-1.0.17-orig/regtest/Makefile.am libsndfile-1.0.17/regtest/Makefile.am
--- libsndfile-1.0.17-orig/regtest/Makefile.am	2008-03-06 01:35:07.000000000 +0000
+++ libsndfile-1.0.17/regtest/Makefile.am	2008-03-06 01:37:27.000000000 +0000
@@ -1,6 +1,8 @@
 ## Process this file with automake to produce Makefile.in
 
+if HAVE_SQLITE3
 bin_PROGRAMS = sndfile-regtest
+endif
 
 noinst_HEADERS = regtest.h
 
